### PR TITLE
Feature anonymous lists

### PR DESCRIPTION
**Pull request by koepferd**
## Proposed Changes

  - added tab view for enquiries to display anonymous enquiries
  - anonymous enquiry view
  - websocket event handling for `newAnonymousEnquiry`

### CODE REVIEW

File: src/components/sessionsList/sessionsList.styles.scss [hunk 1]
```diff
@@ -131,6 +131,38 @@ $headerHeight: 80px;
...
+			&:last-of-type {
+				margin-left: $grid-base-three;
+			}
+
+			.text:hover {
+				color: $primary;
+			}
+		}
+
+		.sessionsList__tabs--active {
```
Comment (iam-robin) [inline src/components/sessionsList/sessionsList.styles.scss]: is there a reason why you didn't used &--active selector here?

File: src/resources/scripts/i18n/de/sessionList.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
 const sessionList = {
 	'teamsession': 'Team Beratung',
 	'empty': 'Momentan sind keine Anfragen vorhanden.',
+	'empty.anonymous':
+		'Aktuell warten keine anonymen Ratsuchenden auf einen Live-Chat.',
 	'preview.headline': 'Erstanfragen',
+	'preview.registered.tab': 'Erstanfragen',
+	'preview.anonymous.tab': 'Live-Chat Anfragen',
```
Comment (iam-robin) [inline src/resources/scripts/i18n/de/sessionList.ts:8]: shouldn't it be "Live-Chat-Anfragen"?

Comment (koepferd) [reply src/resources/scripts/i18n/de/sessionList.ts:8]: No, that is right out of the design specs.

Comment (iam-robin) [reply src/resources/scripts/i18n/de/sessionList.ts:8]: does not mean it is right 😜

Comment (koepferd) [reply src/resources/scripts/i18n/de/sessionList.ts:8]: I know, but concept confirmed the wording 😝

File: src/components/sessionsList/sessionsList.styles.scss (hunk 1 above)
Comment (koepferd) [reply src/components/sessionsList/sessionsList.styles.scss]: nope, thanks, cleaned that up.

**Review by iam-robin — APPROVED** [review]

**Pull request merged**